packages/mui-material/src/RadioGroup/RadioGroup.d.ts

```diff
@@ -31,7 +31,7 @@ export type RadioGroupClassKey = keyof NonNullable<RadioGroupProps['classes']>;
  *
  * Demos:
  *
- * - [Radio buttons](https://mui.com/material-ui/react-radio-button/)
+ * - [Radio button](https://mui.com/material-ui/react-radio-button/)
```

Comment:
```suggestion
 * - [Radio Button](https://mui.com/material-ui/react-radio-button/)
```

Reply:
@samuelsycamore I won't make these changes as this is auto-generated. It's taking the h1 of https://mui.com/material-ui/react-radio-button/.

Reply:
Also, I don't agree on this one, it's the component is Radio, the button is only there to disambiguate.

Reply:
Ah, I didn't realize these were auto-generated. Can we change the way they're generated so that they capitalize every word in the name? Otherwise it's inconsistent.